### PR TITLE
Improve teachers teams UX 

### PR DESCRIPTION
**Pull request by quitterie-lcs**
## Purpose

Some details needed to be improved for users experience.

## Proposal

- [x] set default theme to light
- [x] predefine dashboard uids/remove post deployement hooks
- [x] add home page for teacher team

### CODE REVIEW

**Review by jmaupetit — CHANGES_REQUESTED** [review]
I think you must remove the post-deploy hook script but restore the hooks feature.

**Review by jmaupetit — COMMENTED** [review]
Great! I suggest few interface improvements :muscle:

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,10 +8,23 @@ Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to
 
 ## [Unreleased]
 
+### Added
+
+- Predefined dashboard uids
```
[the remark this answers is not in the document]
Comment (quitterie-lcs) [reply CHANGELOG.md:13]: I would keep this in the `added` section as there was no pre-definition before ? WDYT?

[the remark this answers is not in the document]
Comment (jmaupetit) [reply CHANGELOG.md:13]: Do as you please.

**Review by jmaupetit — APPROVED** [review]

**Pull request merged**